<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pong Menu</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>pong_fon_menu.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>220</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 4px solid rgb(85, 85, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: #40404D;
color: white;
border-radius: 10px;
}

QPushButton:hover{
border: 6px solid rgb(0, 170, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: #40404D;
color: white;
border-radius: 10px;
}

QPushButton:pressed{
border: 6px solid rgb(0, 170, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: rgb(103, 134, 103);
color: white;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Бот: уровень невозможно</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MV Boli&quot;;
color: rgb(255, 85, 0)</string>
   </property>
   <property name="text">
    <string>Игра 1 на 1:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>240</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 4px solid rgb(85, 85, 127);
color: red;
font: 14pt &quot;MV Boli&quot;;
}

QPushButton:hover{
border: 4px solid rgb(0, 170, 255);
color: red;
font: 14pt &quot;MV Boli&quot;;

}

QPushButton:pressed{
border: 4px solid rgb(255, 85, 127);
color: red;
font: 14pt &quot;MV Boli&quot;;

}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>fon2.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>211</width>
     <height>190</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>220</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 4px solid rgb(85, 85, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: #40404D;
color: white;
border-radius: 10px;
}

QPushButton:hover{
border: 6px solid rgb(0, 170, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: #40404D;
color: white;
border-radius: 10px;
}

QPushButton:pressed{
border: 6px solid rgb(0, 170, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: rgb(103, 134, 103);
color: white;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Бот: уровень средне</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MV Boli&quot;;
color: rgb(255, 85, 0)</string>
   </property>
   <property name="text">
    <string>Игра с ботом:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>620</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 4px solid rgb(85, 85, 127);
color: red;
font: 14pt &quot;MV Boli&quot;;
}

QPushButton:hover{
border: 4px solid rgb(0, 170, 255);
color: red;
font: 14pt &quot;MV Boli&quot;;

}

QPushButton:pressed{
border: 4px solid rgb(255, 85, 127);
color: red;
font: 14pt &quot;MV Boli&quot;;

}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>fon4.jpg</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>211</width>
     <height>190</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 4px solid rgb(85, 85, 127);
color: red;
font: 14pt &quot;MV Boli&quot;;
}

QPushButton:hover{
border: 4px solid rgb(0, 170, 255);
color: red;
font: 14pt &quot;MV Boli&quot;;

}

QPushButton:pressed{
border: 4px solid rgb(255, 85, 127);
color: red;
font: 14pt &quot;MV Boli&quot;;

}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="icon">
    <iconset>
     <disabledoff>fon3.jpg</disabledoff>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>211</width>
     <height>190</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 4px solid rgb(85, 85, 127);
color: red;
font: 14pt &quot;MV Boli&quot;;
}

QPushButton:hover{
border: 4px solid rgb(0, 170, 255);
color: red;
font: 14pt &quot;MV Boli&quot;;

}

QPushButton:pressed{
border: 4px solid rgb(255, 85, 127);
color: red;
font: 14pt &quot;MV Boli&quot;;

}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="icon">
    <iconset>
     <disabledon>fon1.jpg</disabledon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>190</width>
     <height>190</height>
    </size>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;MV Boli&quot;;
color: rgb(255, 85, 0)</string>
   </property>
   <property name="text">
    <string>Режим игры:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;MV Boli&quot;;
color: rgb(216, 255, 108);</string>
   </property>
   <property name="text">
    <string>Выберите фон:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>220</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 4px solid rgb(85, 85, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: #40404D;
color: white;
border-radius: 10px;
}

QPushButton:hover{
border: 6px solid rgb(0, 170, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: #40404D;
color: white;
border-radius: 10px;
}

QPushButton:pressed{
border: 6px solid rgb(0, 170, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: rgb(103, 134, 103);
color: white;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>1 на 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>220</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 4px solid rgb(85, 85, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: #40404D;
color: white;
border-radius: 10px;
}

QPushButton:hover{
border: 6px solid rgb(0, 170, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: #40404D;
color: white;
border-radius: 10px;
}

QPushButton:pressed{
border: 6px solid rgb(0, 170, 127);
font: 12pt &quot;MV Boli&quot;;
background-color: rgb(103, 134, 103);
color: white;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Бот: уровень легко</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 127);
font: 20pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string>Счёт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 127);
font: 20pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 127);
font: 20pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 127);
font: 14pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 127);
font: 14pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string>Игрок № 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 127);
font: 14pt &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string>Игрок № 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MV Boli&quot;;
color: rgb(216, 255, 108);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
